ui/app/components/clients/token/monthly-new.ts

```diff
@@ -0,0 +1,51 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: BUSL-1.1
+ */
+
+import Component from '@glimmer/component';
+import { calculateAverage } from 'vault/utils/chart-helpers';
+
+import type { ClientActivityMonthly, ClientActivityTotals } from 'vault/vault/models/clients/activity';
+
+interface Args {
+  byMonthActivityData: ClientActivityMonthly[];
+  authMount: string;
+  runningTotals: ClientActivityTotals;
+  responseTimestamp: string;
+}
+
+export default class ClientsTokenMonthlyNewComponent extends Component<Args> {
+  runningTotalLegend = [
```

Comment:
Follow-on cleanup 🧹 - move this to `token` page component and pass to both MonthlyNew and MonthlyUsage chart. Also I was attempting to keep the legend name the same as the component for traceability, so renaming this to `tokenMonthlyLegend` might be helpful

Reply:
Thanks I added these notes to the cleanup ticket.